doc/lightning-close.7.md

```diff
@@ -28,6 +28,22 @@ The default is 2 days (172800 seconds).
 The *destination* can be of any Bitcoin accepted type, including bech32.
 If it isn't specified, the default is a c-lightning wallet address.
 
+The *fee_negotiation_step* parameter controls how closing fee
+negotiation is performed assuming the peer proposes a fee that is
+different than our estimate. On every negotiation step we must give up
+some amount from our proposal towards the peer's proposal. This parameter
+can be an integer in which case it is interpreted as number of satoshis
+to step at a time. Or it can be an integer followed by "%s" to designate
```

Comment:
"%s"?  Not sure where the 's' came from?

Reply:
[Fixed in a followup PR](https://github.com/ElementsProject/lightning/pull/3629/commits/6502e643b344ea0dda4a8bc2c8e4b3f4bc422ec9). I must have been in C-mode-autopilot when writing the .md file.